.github/workflows/dev-deploy-api.yml

```diff
@@ -95,36 +58,36 @@ jobs:
           bullmq_secret: ${{ secrets.BULL_MQ_PRO_NPM_TOKEN }}
 
       - name: Checkout cloud infra
-        if: ${{ contains (matrix.name,'ee') }}
+        if: ${{ contains (matrix.name,'-ee') }}
```

Comment:
I decided to add `-` in order to be more explicit because in the future we could have some packages with double `e` and some one will merge it to the ci and we could notice it only once it is too late.